src/index.html

```diff
@@ -1,135 +1,139 @@
+  <meta property="og:url"
+    content="https://kompaspedia.kompas.id/baca/paparan-topik/kawasan-ekonomi-khusus-pertumbuhan-ekonomi-baru-dengan-daya-saing-tinggi" />
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css"
+    integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">
+  <script type="module" src="/build/kompas-web-components.esm.js"></script>
+  <script nomodule src="/build/kompas-web-components.js"></script>
+  <style>
+    body {
+      font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    }
+
+    h2 {
+      font-family: 'Times New Roman', Times, serif;
+      font-size: 36px;
+    }
+
+    p {
+      margin-bottom: 16px;
+      text-align: left;
+    }
+
+    #app {
+      font-family: Avenir, Helvetica, Arial, sans-serif;
+      -webkit-font-smoothing: antialiased;
+      -moz-osx-font-smoothing: grayscale;
+      text-align: center;
+      color: #2c3e50;
+      margin-top: 60px;
+    }
+
+  </style>
+  <!-- Google Tag Manager -->
+  <script>
+    (function (w, d, s, l, i) {
+      w[l] = w[l] || [];
+      w[l].push({
+        'gtm.start': new Date().getTime(),
+        event: 'gtm.js'
+      });
+      var f = d.getElementsByTagName(s)[0],
+        j = d.createElement(s),
+        dl = l != 'dataLayer' ? '&l=' + l : '';
+      j.async = true;
+      j.src =
+        'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
+      f.parentNode.insertBefore(j, f);
+    })(window, document, 'script', 'dataLayer');
+  </script>
+  <!-- End Google Tag Manager -->
+</head>
+
+<body id="app">
+  <div style="max-width: 1165px;margin:auto">
+    <header style="width:100%;position:fixed;left:0;top:0px;background-color: #0356A8;padding:0.75rem;z-index: 10;">
+      <div style="max-width: 1280px;margin:auto;display: flex">
+        <kompas-header-account
+          cart-url="/cart"
+          logout-url="/logout"
+          manage-account-url="/manage-count"
+          notification-url="/notif"
+          notification-total=10
+          orders-url="/orders"
+          sidebar-top-spacing=48
+          subscription-url=""
+          style="margin-left: auto;"
+        />
+      </div>
+    </header>
+    <h2>Lipsum Dei</h2>
+    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Adipisci similique omnis dolorem dolores aliquam quis
+      perspiciatis fugiat illum nisi laboriosam accusantium quam explicabo suscipit, asperiores blanditiis, ipsum
+      exercitationem, reiciendis vitae!
+    </p>
+    <!-- <kompas-paywall type="reguler" is-login="true" is-with-header="true" text-header="this is message for header information" user-guid="111000" subscription-status="register" >
+    </kompas-paywall> -->
+    <div style="position:relative">
+      <img src="https://i.ibb.co/hg3vr2T/bg-kompas-subscription-mobile.png" style="width: 100%; height: 100%; object-fit: cover;" alt="bg-kompas-subscription-mobile" />

```

Comment:
image not found, please replace to this
```suggestion
      <img src="https://dmm0a91a1r04e.cloudfront.net/JTYT2c26Hx4LmglA652oauNuViI=/1280x545/https%3A%2F%2Fasset.kgnewsroom.com%2Fphoto%2Fpre%2F2021%2F11%2F23%2Fa59d565e-def1-4efc-8790-2969a8b949d8_jpeg.jpg" style="width: 100%; height: 100%; object-fit: cover;" alt="bg-kompas-subscription-mobile" />

```